ccdproc/combiner.py

```diff
@@ -177,6 +177,58 @@ def scaling(self, value):
             # reshape so that broadcasting occurs properly
             self._scaling = self.scaling[:, np.newaxis, np.newaxis]
 
+    # set up IRAF-like minmax clipping
+    def iraf_minmax_clipping(self, nlow=None, nhigh=None):
```

Comment:
would default values of 1 for these make more sense? That would also eliminate the `if`s below that check for `None`.